From a4c4365e66c77410ec50af0c64cac8e37088830f Mon Sep 17 00:00:00 2001
From: David Daney <david.daney@cavium.com>
Date: Sun, 19 May 2019 12:06:43 +0200
Subject: [PATCH 121/345] MIPS: OCTEON: Change load address to waste less
 memory.

"Modern" OCTEON bootloaders allow us to load the image at 0x100000.
Moving there from 0x1100000 makes 16MB more memory usable.

Signed-off-by: David Daney <david.daney@cavium.com>
---
 arch/mips/cavium-octeon/Platform | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/cavium-octeon/Platform b/arch/mips/cavium-octeon/Platform
index 45be853700e6..a95d7d19b45c 100644
--- a/arch/mips/cavium-octeon/Platform
+++ b/arch/mips/cavium-octeon/Platform
@@ -4,4 +4,4 @@
 platform-$(CONFIG_CAVIUM_OCTEON_SOC)	+= cavium-octeon/
 cflags-$(CONFIG_CAVIUM_OCTEON_SOC)	+=				\
 		-I$(srctree)/arch/mips/include/asm/mach-cavium-octeon
-load-$(CONFIG_CAVIUM_OCTEON_SOC)	+= 0xffffffff81100000
+load-$(CONFIG_CAVIUM_OCTEON_SOC)	+= 0xffffffff80100000
-- 
2.25.1

